modular-docs-manual/content/topics/module_mod-docs-assembly-examples.adoc

```diff
@@ -5,4 +5,4 @@ link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_ma
 
 link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html-single/designing_a_decision_service_using_spreadsheet_decision_tables/index[Designing a decision service using spreadsheet decision tables]
 
-link:https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/1.3/html-single/getting_started_with_red_hat_build_of_quarkus/index[Getting started with Red Hat Build of Quarkus]
+link:https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/quarkus-3.2/guide/00b48021-40c0-40d8-95e6-64292948cf6f[Getting started with Red Hat Build of Quarkus]
```

Comment:
The Quarkus docs team is migrating this content from Pantheon II to Pantheon (I), which will affect this URL. Could you remove or replace this with an example from another product?

Reply:
@rolfedh Thank you. That URL was suspicious to me, but the content seemed to be a good example. I'll stay on the safe side though, and I will follow your advice.